src/backend/distributed/sql/citus--8.3-1--9.0-1.sql

```diff
@@ -174,4 +174,41 @@ COMMENT ON FUNCTION worker_apply_sequence_command(text,regtype)
 
 #include "udfs/citus_isolation_test_session_is_blocked/9.0-1.sql"
 
+
+CREATE FUNCTION ensure_truncate_trigger_is_after()
+    RETURNS void
+    LANGUAGE plpgsql
+    SET search_path = pg_catalog
+    AS $$
+DECLARE
+    table_name regclass;
+    command text;
+    trigger_name text;
+BEGIN
+    --
+    -- register triggers
+    --
+    FOR table_name, trigger_name IN SELECT tgrelid::regclass, tgname
+      FROM pg_dist_partition
+      JOIN pg_trigger ON tgrelid=logicalrelid
+      JOIN pg_class ON pg_class.oid=logicalrelid
+      WHERE
+        tgname LIKE 'truncate_trigger_%' AND tgfoid = 'citus_truncate_trigger'::regproc
+    LOOP
+        command := 'drop trigger ' || trigger_name || ' on ' || table_name;
+        EXECUTE command;
+        command := 'create trigger ' || trigger_name || ' after truncate on ' || table_name || ' execute procedure pg_catalog.citus_truncate_trigger()';
+        EXECUTE command;
+        command := 'update pg_trigger set tgisinternal = true where tgname = ' || quote_literal(trigger_name);
+        EXECUTE command;
+    END LOOP;
+END;
+$$;
+
+SELECT ensure_truncate_trigger_is_after();
+DROP FUNCTION ensure_truncate_trigger_is_after;
+
+-- This sequence is unused
+DROP SEQUENCE pg_catalog.pg_dist_jobid_seq;
```

Comment:
@jeltef this is indeed unused on community. Otoh, there are some references to this sequence in Enterprise tests, and that's why merges to Enterprise fail now.

It is easy to fix, but it took a bit of my time to understand what is going on. I will update those tests first thing in the morning.

My point is that it is a really nice practice to merge changes into Enterprise after merging your PR in community. No matter how small and simple your change is.